src/account.ts

```diff
@@ -70,6 +80,7 @@ export class Account {
      */
     async state(): Promise<AccountState> {
         await this.ready;
+        this._state.balance = await this.getAccountBalance();
```

Comment:
let's not do this, `state()` is called in many places, better avoid slowing it down by making additional query every time

Reply:
Good point.

Reply:
Changes are done.

It might be a bit more efficient if we add optional param to `state()` that will decide if the balance should be calculated and added to `_state`. With current changes, we need to call `state()` again in `getAccountBalance` to calculate balance, so for example, in Wallet we will call `state()` then call `getAccountBalance` that will again call `state()`. What do you think?